src/snowflake/cli/api/project/schemas/project_definition.py

```diff
@@ -209,11 +212,13 @@ def _merge_mixins_with_entity(
         for mixin_name in entity_mixins_names:
             if mixin_name not in mixin_defs:
                 raise ValueError(f"Mixin {mixin_name} not defined")
-
+        context = get_cli_context()
```

Comment:
Why we need context here?

Reply:
Leftover from previous attempts, already removed